src/notifications/use-push-notifications.tsx

```diff
@@ -53,23 +53,31 @@ export const usePushNotifications = () => {
   }, []);
 
   const register = useCallback(async () => {
+    const token = await messaging().getToken();
+    if (!token) return;
+    registerMutation.mutate({token, language: language});
+    return token;
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [language, registerMutation.mutate]);
```

Comment:
exhaustive-deps wanted me to add `registerMutation` instead of `registerMutation.mutate`, but that would call on every render.

Reply:
Yeah, I feel like exhaustive deps could be more forgiving and allow `registerMutation.mutate` as it does with other non-function fields.

But the answer is to destructure `registerMutation` and use only `mutate` (maybe with a better name) in the `useEffect`.

Reply:
Ah, of course 💡